bootstrap.php

```diff
@@ -1319,7 +1319,7 @@ function mailchimp_member_data_update($user_email = null, $language = null, $cal
  * @param string $samesite
  * @return void
  */
-function mailchimp_set_cookie($name, $value, $expire, $path, $domain = '', $secure = true, $httponly = false, $samesite = 'Strict') {
+function mailchimp_set_cookie($name, $value, $expire, $path, $domain = '', $secure = false, $httponly = false, $samesite = 'Strict') {
```

Comment:
@alejandrogiv can you revert this back to `$secure = true` - we don't do this in production - so it would be a localhost only update. If we need to do something like adding a PHP CONSTANT to do a default - we can do this.

Reply:
so weird I was sure that I took it back, my bad doing now!